packages/bridge-controller/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Bump `@metamask/controller-utils` to `^11.8.0` ([#5765](https://github.com/MetaMask/core/pull/5765))
+- Added `isUnifiedUIEnabled` flag to `ChainConfiguration` feature-flag type and update validators accordingly ([#5783](https://github.com/MetaMask/core/pull/5783))
```

Comment:
Can we put this under Added Section instead ? Also I only see a new added property where did we update validators?